### ISSUE TITLE
Allow custom command path to be specified

### ISSUE THREAD

**Issue by mike-potter** [docksal/docksal#846]
Currently, Docksal looks within the .docksal/commands (or ~/.docksal/commands) folder for custom command scripts.  Because Docksal has the *awesome* ability to recognize the 

```
#: exec_target = cli
```

comment at the top of a script file, these custom commands can also be executed on other non-Docksal (CI) environments.  For example, we have a "build" command that we also use on CI to run "composer install, config-import, clear-cache, etc".

However, we are not comfortable executing a script within the .docksal/commands folder on CI because we only want to use docksal locally and would be deleting the .docksal folder on CI/Stage/Prod/etc.

It would be super useful if there was a configuration path to tell Docksal where to look for the custom command folder.

As a current workaround, we have our scripts in a top-level /scripts folder.  Then for each script we create a custom command script in .docksal/commands that just executes the corresponding script in the /scripts folder.  If would be nice to instead just tell Docksal to look within /scripts directly.

Any chance of getting this simple config request?  It would hugely simplify our project scripting and help encourage even more use of Docksal locally.

**Comment by achekulaev** [comment]
Docksal looks for custom commands in 2 places*: 
- `<project>/.docksal/commands` 
- `$HOME/.docksal/commands`. 

It seems like the second location should work for your CI needs.

--

\* - actually in 4, since the same applies to addons

**Comment by mike-potter** [comment]
No, these are still project-level scripts, not global.  The $HOME/.docksal folder doesn't exist in our CI environment.  We just have the files that are in the repo.  So right now we have /scripts and /.docksal but there is a move to remove the .docksal folder on CI and above environments since they don't need it.

**Comment by achekulaev** [comment]
Why don't you simply make your `<project>/docksal/commands` a symlink to your scripts folder then?

**Comment by mike-potter** [comment]
We might do that.  However, we *do* have a couple of custom commands that are purely for local dev and docksal specific.  So we have those docksal-specific commands in ``<project>/.docksal/commands`` but have the non-specific commands (like rebuild) in ``<project>/scripts``.  In the CI environments it only needs the stuff in ``scripts``

**Comment by lmakarov** [comment]
@mike-potter how about symlinking?

**Comment by achekulaev** [comment]
@lmakarov fin will not work with symlinks for commands or command folder.

**Comment by achekulaev** [comment]
@mike-potter the change like this is not too simple, because the way fin looks for commands is hardcoded. This was never meant to be configurable so changing it is not as easy as it might seem. It also seems a bit against the grind to make this configurable. 

What I can propose is that I can improve the way `fin` runs commands, so that it would accept symlink for a command file. This way your could have your docksal-specific commands playing along with symlinks to commands in the `<project>/scripts` folder

```
<project>/
  .docksal/
    commands/
      docksal-specific-command
      linked-command -> ../../scripts/somescript
  scripts/
    somescript
```

**Comment by mike-potter** [comment]
Well, add this as a "feature request".  Making Docksal more configurable like this is always useful.  Paths shouldn't be hard coded. It already looks in two different places so didn't think looking in a third place would be much of an issue.  Maybe I'll take a look at the "fin" bash script and see about making a PR at some point.

The symlink approach really isn't that different from what I have now where the script in ``<project/.docksal/commands`` just executes the script in ``<project>/scripts``.

**Comment by lmakarov** [comment]
> The symlink approach really isn't that different from what I have now where the script in `<project/.docksal/commands` just executes the script in `<project>/scripts`.

@mike-potter any issues with this approach?

**Comment by mike-potter** [comment]
Well that's why I requested the feature...this approach requires me to have duplicate scripts within `<project>/.docksal/commands` for every script that I have in `<project>/scripts`.  So just an annoyance for devs on a project to create a script in 2 places each time they want to add something.

**Comment by jhedstrom** [comment]
It should be noting that while symlinking allows custom commands to be executed, they are not listed in the _Custom commands:_ output of the `fin` command.

<img width="724" alt="Missing custom commands screenshot" src="https://user-images.githubusercontent.com/76833/56322351-b0394580-611d-11e9-90c9-5bcf648aec3a.png">

I'm looking into that now.
